README.md

```diff
@@ -18,14 +18,16 @@ For older versions of Rails, use these branches of @mattbrictson's original repo
 - [Rails 5.0.x](https://github.com/mattbrictson/rails-template/tree/rails-50)
 - [Rails 5.1.x](https://github.com/mattbrictson/rails-template/tree/rails-51)
 - [Rails 5.2.x](https://github.com/mattbrictson/rails-template/tree/rails-52)
+- [Rails 6.0.x](https://github.com/mattbrictson/rails-template/tree/rails-60)
+- [Rails 6.1.x](https://github.com/mattbrictson/rails-template/tree/rails-61)
```

Comment:
I think we should remove these links completely tbh. This repo is always about the most recent version of Rails and linking off to Matt's repo will just confuse people because we have diverged so much from his repo.

Reply:
Ok done :)